src/modules/app/createReduxStore.js

```diff
@@ -1,54 +1,49 @@
 import { applyMiddleware, combineReducers, compose, createStore } from 'redux'
 import thunkMiddleware from 'redux-thunk'
-import { initializeCurrentLocation, routerForBrowser } from 'redux-little-router'
+import { connectRoutes } from 'redux-first-router'
 import { createLogger } from 'redux-logger'
 
-import RouteConfig from 'modules/app/RouteConfig'
-import endpointReducers from 'modules/endpoint/reducers'
-import setLanguageChangeUrlsReducer from '../language/reducers/setLanguageChangeUrls'
+import endpointReducers from '../endpoint/reducers'
 import { createResponsiveStateReducer, responsiveStoreEnhancer } from 'redux-responsive'
+import defaultRoutesMap from './routesMap'
 
-const createReduxStore = (createHistory, initialState = {}, routes = new RouteConfig()) => {
+import onBeforeChange from './onBeforeChange'
+
+/**
+ * Creates our Redux store
```

Comment:
I think we can remove this comment. The name createReduxStore is already expressive enough :)